types/smallwei__avue/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for avue 3.0
+// Project: https://github.com/nmxiaowei/avue
+// Definitions by: Lyoko <https://github.com/SoulLyoko>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference lib="dom" />
+/// <reference path="./element-plus.d.ts" />
+/// <reference path="./dic.d.ts" />
+/// <reference path="./form.d.ts" />
+/// <reference path="./crud.d.ts" />
+/// <reference path="./tree.d.ts" />
+/// <reference path="./global.d.ts" />
+
+// import { App } from 'vue';
+
+interface AvueConfig {
+    size?: 'large' | 'default' | 'small';
+    menuType?: 'button' | 'icon' | 'text' | 'menu';
+    theme?: 'dark';
+    axios?: any;
+}
+interface AvueInstaller {
+    install(app: any, config?: AvueConfig): void;
+}
+declare const Avue: AvueInstaller;
+
+export default Avue;
```

Comment:
the module seems to be webpack exported UMD module:
https://gitee.com/smallweigit/avue/blob/master/build/config.js#L66
https://unpkg.com/browse/@smallwei/avue@2.10.0/lib/avue.js
(L:7 and others)


https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library